docs/running-on-kubernetes.md

```diff
@@ -400,7 +415,6 @@ There are several Spark on Kubernetes features that are currently being worked o
 Some of these include:
 
 * Dynamic Resource Allocation and External Shuffle Service
-* Local File Dependency Management
```

Comment:
Out of curiosity, does any s3 object store fit the HCFS category?

Reply:
A quick example is [minio](https://github.com/minio/cookbook/blob/master/docs/apache-spark-with-minio.md).

Reply:
We have environments where there is nothing remotely HDFS like available and the systems are typically air-gapped so using external services like S3 isn't an option either.  Primary storage is usually a high performance parallel file system (Lustre or IBM Spectrum Scale) which is just a POSIX compliant file system mounted to all nodes over the system interconnect.

Using `hostPath` volume mounts isn't a realistic option either because these environments have strict security requirements.

Reply:
In my opinion we need both options: a) upload to some dfs/obect storeservice b) a file server.
We cannot possibly support all systems out there if they have their own client libs.